specification/botservice/resource-manager/Microsoft.BotService/stable/2021-03-01/botservice.json

```diff
@@ -0,0 +1,2640 @@
+        },
+        "description": "Get a Connection Setting registration for a Bot Service",
+        "operationId": "BotConnection_Get",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/connectionNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The resource provider should return 200 (OK) to indicate that the operation completed successfully. ",
+            "schema": {
+              "$ref": "#/definitions/ConnectionSetting"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed. If the resource group *or* resource does not exist, 404 (NotFound) should be returned.",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "BotConnection"
+        ],
+        "description": "Deletes a Connection Setting registration for a Bot Service",
+        "operationId": "BotConnection_Delete",
+        "x-ms-examples": {
+          "Update Connection Setting": {
+            "$ref": "./examples/DeleteConnection.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/connectionNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "A 200 (OK) should be returned if the object exists and was deleted successfully;"
+          },
+          "204": {
+            "description": "a 204 (NoContent) should be used if the resource does not exist and the request is well formed."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.BotService/botServices/{resourceName}/connections": {
+      "get": {
+        "tags": [
+          "BotConnection"
+        ],
+        "description": "Returns all the Connection Settings registered to a particular BotService resource",
+        "operationId": "BotConnection_ListByBotService",
```

Comment:
Operation Group should be plural (BotConnections), again, no breaking changes, though

Reply:
Keep as is for back-compat. since according to https://swagger.io/docs/specification/paths-and-operations/ "Some code generators use this value to name the corresponding methods in code."